gen/definitions/vlan_tag.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: VLAN Tag
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/vlantags
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    description: User-created name of the resource.
+    mandatory: true
+    example: vlan_tag_1
+  - model_name: description
+    type: String
+    description: Optional user-created description.
+    example: "My TAG id"
+    mandatory: true
+  - model_name: overridable
+    type: Bool
+    description: Indicates whether object values can be overridden.
+    exclude_example: true
+    test_value: "true"
+    minimum_test_value: "true"
+  - model_name: type
+    type: String
+    mandatory: true
+    example: "VlanTag"
+  - model_name: startTag
+    type: String
+    example: "12"
+    test_value: "12"
+    data_path: ["data"]
+  - model_name: endTag
```

Comment:
Not sure if this one should be mandatory as well.

Reply:
Done